sycl/test-e2e/GroupAlgorithm/SYCL2020/reduce_over_group_size.cpp

```diff
@@ -1,8 +1,8 @@
-// Test hangs on AMD with https://github.com/intel/llvm/pull/8412
-// UNSUPPORTED: hip_amd
-
 // Windows doesn't yet have full shutdown().
 // UNSUPPORTED: ze_debug && windows
+//
+// The CI GPU doesn't report support for shared USM
+// XFAIL: hip_amd
```

Comment:
```suggestion
// REQUIRES: aspect-usm_shared_allocations
```
That should make it so other implementations that don't support shared USM are skipped automatically.